spec/core_functions/meta/apply.hrx

```diff
@@ -0,0 +1,122 @@
+<===> options.yml
+---
+:todo:
+- sass/libsass#2807
+
+<===>
+================================================================================
+<===> use-as-function/input.scss
```

Comment:
[Put error specs in a separate "error" directory](https://github.com/sass/sass-spec/blob/main/STYLE_GUIDE.md#do-put-error-specs-in-a-separate-error-directory)